src/store/addressbooks.js

```diff
@@ -59,7 +59,28 @@ export function mapDavCollectionToAddressbook(addressbook) {
 		owner: addressbook.owner,
 		readOnly: addressbook.readOnly === true,
 		url: addressbook.url,
-		dav: addressbook
+		dav: addressbook,
+		shares: addressbook.shares.map(sharee => Object.assign({}, mapDavShareeToSharee(sharee)))
+	}
+}
+
+/**
+ * map a dav collection to our addressbook object model
+ *
+ * @param {Object} sharee the sharee object from the cdav library shares
+ * @returns {Object}
+ */
+export function mapDavShareeToSharee(sharee) {
+	const id = sharee.href.split('/').slice(-1)[0]
+	const name = sharee['common-name']
+		? sharee['common-name']
+		: id
+	return {
+		displayName: name,
+		id: id,
+		writeable: sharee.access[0].endsWith('read-write'),
+		isGroup: sharee.href.indexOf('groups') > -1,
```

Comment:
`sharee.href.indexOf('principal:principals/groups/') === 0`
Otherwise it will also be true if the username contains `groups` for whatever reason.